package.json

```diff
@@ -69,6 +69,7 @@
     "google-gax": "^0.18.0",
     "is": "^3.2.1",
     "lodash.merge": "^4.6.1",
+    "mathjs": "^5.0.4",
```

Comment:
does this belong here or in the samples `package.json`?

Reply:
removed!  had some issues with eslint-config-prettier (applies to last 3, sorry about that!)